src/debugger/Debugger.js

```diff
@@ -129,23 +129,23 @@ export class DebugServer {
     switch (command) {
       case DebugMessage.BREAKPOINT_ADD_COMMAND:
         invariant(args.kind === "breakpoint");
-        this._breakpoints.addBreakpoint(args.filePath, args.line, args.column);
-        this._channel.sendBreakpointAcknowledge(DebugMessage.BREAKPOINT_ADD_ACKNOWLEDGE, requestID, args);
+        this._breakpoints.addBreakpointMulti(args.breakpoints);
```

Comment:
If you follow the same suggestion as _processPrepackMessage() here by not splitting requestID from the request, you can collapse the remaining marshalXXX() call into a single marshal() API too. Feel free to do in later PR though.